modules/authorization/src/controllers/relations.controller.ts

```diff
@@ -58,6 +58,69 @@ export class RelationsController {
     return relationResource;
   }
 
+  async checkRelations(subject: string, relation: string, resources: string[]) {
+    checkRelation(subject, relation, resources[0]);
+    const computedTuples = resources.map(r => computeRelationTuple(subject, relation, r));
+    const relationResources = await Relationship.getInstance().findMany({
+      computedTuple: { $in: computedTuples },
+    });
+    if (relationResources.length) throw new Error('Relations already exists');
```

Comment:
This is probably gonna get rebased anyway, but simply ignore existing relations instead.
If you'd rather stick with whatever's already in `main`, fix singular/plural typo.